var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py

```diff
@@ -12,5 +12,4 @@ class NeurodamusHippocampus(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
 
     version('develop', git=git, branch='master', submodules=True)
-    version('1.0', tag='1.0', submodules=True)
```

Comment:
If tests are passing, we should tag this as version...eg 0.1?

Reply:
I would tag only when all tests are passing and we solved the unknown things. Besides now that we use LC_ALL='C' we will need changes either to recipe or results

Reply:
Ok!